.evergreen/config.yml

```diff
@@ -579,6 +613,21 @@ tasks:
             MONGODB_VERSION: "latest"
             TOPOLOGY: "sharded_cluster"
         - func: "run tests"
+  
+    - name: "test-latest-load_balancer"
+      tags: ["latest", "load-balancer"]
+      commands:
+        - func: "prepare resources"
+        - func: "fix absolute paths"
+        - func: "bootstrap mongo-orchestration"
+          vars:
+            MONGODB_VERSION: "latest"
```

Comment:
technically, we are supposed to test against "each server version that supports load balanced clusters". 

this is kind of weird now because no servers actually support it, yet, with the serviceId mocking support they all appear to. 

for now, I opted to just test against latest since that's the closest server version to actual support, but I suppose once 5.1 comes out with serviceId support (and we get a "test drivers against 5.1" ticket) we can start testing against both 5.1 and latest.